backend/wmg/cube_creation/load_cube.py

```diff
@@ -201,6 +202,10 @@ def load_data_and_create_cube(path_to_datasets, group_name):
         create_cube(group_name)
     except Exception as e:
         logger.error(f"Issue creating the cube: {e}")
+    try:
+        create_cell_count_cube(group_name)
+    except Exception as e:
+        logger.error(f"Issue creating the cell count cube: {e}")
```

Comment:
beyond the scope of this PR, but is there any point in running the rest of the pipeline if either of the 2 cube creations fail? shouldn't we rethrow errors and fail loudly?

Reply:
That's a good point, it's probably worth removing the try/except here and let the pipeline fail. Not sure that the "naive" batch implementation allows for retries, but given that this only needs to run once per week, it's probably enough if we  have some kind of alerting in place so it can be retried manually.